VirtoCommerce.Platform.Core/Notifications/EmailNotification.cs

```diff
@@ -13,6 +9,6 @@ protected EmailNotification(IEmailNotificationSendingGateway gateway) : base(gat
         }
 
         public IList<string> CC { get; set; } = new List<string>();
-        public IList<string> Bcc { get; set; } = new List<string>();
+        public IList<string> BCC { get; set; } = new List<string>();
```

Comment:
Undo.
By convention, two-letter acronyms use all uppercase letters, and acronyms of three or more characters use Pascal casing.
https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1709-identifiers-should-be-cased-correctly?view=vs-2019